CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.17.3]
+### Fixed
+- Stop emitting Dapp Viewed events on Firefox ([#32583](https://github.com/MetaMask/metamask-extension/pull/32583))
+- Make phishing warning events anonymous ([#32635](https://github.com/MetaMask/metamask-extension/pull/32635))
+- Keep showing the approve row in the batch transaction confirmation ([#32658](https://github.com/MetaMask/metamask-extension/pull/32658))
```

Comment:
Nit: "keep showing" makes it unclear exactly what was fixed. I've attempted to re-word it to make it more clear that it's fixing a bug where the approve row disappears briefly.
```suggestion
- Fix momentary disappearance of approve row in batch transaction confirmation ([#32658](https://github.com/MetaMask/metamask-extension/pull/32658))
```

Reply:
Makes sense, thanks for the suggestion !